src/components/TopBar/CallButton.vue

```diff
@@ -181,6 +179,10 @@ export default {
 			return t('spreed', 'Leave call')
 		},
 
+		leaveCallCombinedLabel() {
+			return this.leaveCallLabel + ' ▼'
```

Comment:
We can not have a second icon on the button, so for now I went with this plain text symbol.